__mocks__/@nebula.js/stardust.js

```diff
@@ -1,8 +1,14 @@
 const { useState, useMemo, useEffect } = require("react");
 
 const usePromise = (callback) => {
-  callback();
-  return [Promise.resolve(), undefined];
+  const [result, setResult] = useState();
```

Comment:
I have not looked at the actual `usePromise` code in nebula, but guessing it works something like this. So made sense to have it behave the similar in our tests.